scripts/release-notes.js

```diff
@@ -53,7 +53,9 @@ async function getCommits() {
       const jsonReady = `[${commitsArray
         .map(rawCommitText => {
           return rawCommitText
+            .replace(/	/g, ' ')
```

Comment:
I think it is better to replace the tab character with **2 spaces** to preserve the visual formatting of the text.